src/utils/cleanUpReportHTML.js

```diff
@@ -42,8 +42,8 @@ export function cleanUp(node) {
   });
 
   removeAttributeEls.forEach((el) => {
-    el.removeAttribute("id");
-    el.removeAttribute("class");
+    // el.removeAttribute("id");
```

Comment:
Why is this commented out? Fine with commenting out in the commit, but curious why.

Reply:
That code removes IDs and classes when it exported the HTML. We need those for our anchor links so I commented it out. We might use the cleanup code to do other cleanup if needed.